webpack.config.js

```diff
@@ -0,0 +1,31 @@
+const webpack = require("webpack");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const { CleanWebpackPlugin } = require("clean-webpack-plugin");
+const ZipPlugin = require("zip-webpack-plugin");
+const path = require("path");
+
+const package = require("./package");
+const widgetName = package.name;
+const widgetVersion = package.version;
+
+module.exports = {
+    entry: {
+        [widgetName]: [ `./src/${widgetName}/widget/${widgetName}.js` ]
+    },
+    output: {
+        path: path.resolve(__dirname, "dist/tmp/src"),
+        filename: `${widgetName}/widget/[name].js`,
+        chunkFilename: `${widgetName}/widget/${widgetName}[id].js`,
+        libraryTarget: "amd",
+        publicPath: "/widgets/"
+    },
+    devtool: false,
+    mode: "development",
```

Comment:
Should this be `development` since `webpack.config.dev.js` already overrides this config with the same field?

Reply:
Fixed, my bad. I forgot to change back during tests